src/sg/gov/tech/notarise/1.0/schema.json

```diff
@@ -25,6 +25,11 @@
           "description": "URI to the stored document",
           "type": "string",
           "format": "uri"
+        },
+        "offlineData": {
```

Comment:
the fact that it works offline is just a consequence of the how the data is built. The name here is meaningless and doesn't help to understand what it is.

If you need to add another data for USA that works offline too you will name it offlineData2?

name it according to what *it is* => eu health certificate